test/pkgs_signed/SignedPackage2/pkg.json

```diff
@@ -0,0 +1 @@
+{"license":"","file_hash":null,"name":"SignedPackage","version":"1.0.0","description":"original package","group":"","keywords":null,"dependencies":[],"contents":"","engine_version":"2.1.0.7840","engine":"dynamo","engine_metadata":"","site_url":"","repository_url":"","contains_binaries":true,"node_libraries":["SignedPackage2, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null"]}
```

Comment:
I've already added this package here:
https://github.com/DynamoDS/Dynamo/tree/master/test/builtinpackages%20testdir/Packages/SignedPackage2

perhaps you could make a new signed package so we don't encounter conflicts between the loaded binaries in tests.

Reply:
is the package that you added supposed to be the same as the built-in package ? (only not in a built in location ?)

Reply:
The tests I added which used this package override the builtin packages location to be `test/builtinpackages testdir/`

Reply:
A conflict is what I am trying to test here.
I want the same package to be in both a Built-in localtion and a non built in location.
I want to test that both packages will be shown in the UI but only one will be loaded in the VM

Reply:
Are you suggesting that I should move the duplicate, non-built in package (`SignedPackage2`) to a new location...so that other existing tests do not get conflicts ?

Reply:
ah, I didn't realize you wanted a conflict...